sql/core/src/test/scala/org/apache/spark/sql/SQLQueryTestSuite.scala

```diff
@@ -388,49 +390,52 @@ class SQLQueryTestSuite extends QueryTest with SharedSparkSession {
     }
   }
 
-  /** Executes a query and returns the result as (schema of the output, normalized output). */
-  private def getNormalizedResult(session: SparkSession, sql: String): (StructType, Seq[String]) = {
-    // Returns true if the plan is supposed to be sorted.
-    def isSorted(plan: LogicalPlan): Boolean = plan match {
-      case _: Join | _: Aggregate | _: Generate | _: Sample | _: Distinct => false
-      case _: DescribeCommandBase
-          | _: DescribeColumnCommand
-          | _: DescribeTableStatement
-          | _: DescribeColumnStatement => true
-      case PhysicalOperation(_, _, Sort(_, true, _)) => true
-      case _ => plan.children.iterator.exists(isSorted)
-    }
-
+  protected def handleExceptions(result: => (String, Seq[String])): (String, Seq[String]) = {
```

Comment:
Could you add a function description because we override this differently?
- SQLQueryTestSuite seems to return `(struct<>, ...)`
- ThriftServerQueryTestSuite seems to return `("", answer.sorted)`

Reply:
No, both returns a `(String, Seq[String])` tuple where the first is the schema and the second is the result. Since it's impossible to get the exact spark schema back from a `java.sql.ResultSet` we use empty string in `ThriftServerQueryTestSuite`.

Reply:
I've added a description to it and to its override.